core/domain/topic_domain.py

```diff
@@ -74,9 +77,30 @@ def __init__(self, change_dict):
             self.property_name = change_dict['property_name']
             self.new_value = change_dict['new_value']
             self.old_value = change_dict['old_value']
+        elif self.cmd == CMD_CREATE_NEW:
+            self.name = change_dict['name']
         else:
             raise Exception('Invalid change_dict: %s' % change_dict)
 
+    def to_dict(self):
```

Comment:
Just a reminder to file an issue (for longer-term codebase maintenance) to convert all change dicts to domain objects throughout the codebase, and enumerate the places where this needs to occur, similar to what Nithesh did in #4995. Thanks!

Reply:
Ok, will do that.